manifests/params.pp

```diff
@@ -203,9 +203,15 @@
   # this may be disabled to let a dedicated sudo module manage it instead.
   $manage_sudoersd = true
 
-  # Add a file to /etc/sudoers.d (true) or uses augeas (false)
+  # Setting both $use_sudoersd and $use_sudoers to false means this module will not
+  # touch any sudoers entries. Setting both to true will result in sudoersd winning.
+  # Add a file to /etc/sudoers.d (true).
   $use_sudoersd = true
 
+  # Add contents to /etc/sudoers (true). Defaults to false.
+  $use_sudoers = false
```

Comment:
This should default to `true`, so that the current behaviour of setting $use_sudoersd to false still works as intended. A default of false will change that so it suddenly no longer adds any entry. Both should be disabled manually to get your preferred behaviour of no sudoers management.

Nitpick: delete the second blank line below here too.